artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/AmqpCoreConverter.java

```diff
@@ -264,8 +266,8 @@ protected static ServerJMSMessage processHeader(ServerJMSMessage jms, Header hea
             jms.setLongProperty("JMSXDeliveryCount", header.getDeliveryCount().longValue() + 1);
          }
       } else {
-         jms.setJMSPriority((byte) javax.jms.Message.DEFAULT_PRIORITY);
-         jms.setJMSDeliveryMode(DeliveryMode.NON_PERSISTENT);
+         jms.setJMSPriority((byte) MESSAGE_DEFAULT_PRIORITY);
```

Comment:
I dont understand the driver here. This is handling JMS specific stuff, so it just makes sense to use the api values and not be making our own

Reply:
The AMQP Protocol implementation is depending on the JMS implementation. Going forward we have Jakarta and JMS (at the point they are a transformation). There makes no sense to force a direct dependency on it.

Wildfly is having to import JMS just because of this.

Reply:
So why not simply use the jakata interfaces in server going forwards

Reply:
Its 1..1 afaik atm

Reply:
I'm not sure that we should use JMS or Jakarta Messaging API to convert between core and amqp messsages